projects/frontegg-app/src/lib/frontegg-entitlements.service.ts

```diff
@@ -21,16 +18,16 @@ import { FronteggAppService } from './frontegg-app.service';
   providedIn: 'root',
 })
 export class FronteggEntitlementsService {
-  private entitlementsStateSubject = new BehaviorSubject<any>(undefined);
+  private userStateSubject = new BehaviorSubject<any>(undefined);
```

Comment:
maybe worth passing type instead of any?

Reply:
you are right, we passed a type before, but make admin-box pipeline pass (because of angular tests) I changed it to any. We should change it to the new type when we will remove the FF. Currently any supports v1 and v2